filc/src/runtime.c

```diff
@@ -589,12 +589,12 @@ long zsys_syscall(long n, ...)
         callee = zsys_keyctl;
         break;
 
-    case 203 /* SYS_sched_getaffinity */:
-        callee = zsys_sched_getaffinity;
+    case 203 /* SYS_sched_setaffinity */:
+        callee = zsys_sched_setaffinity;
         break;
 
-    case 204 /* SYS_sched_setaffinity */:
-        callee = zsys_sched_setaffinity;
+    case 204 /* SYS_sched_getaffinity */:
+        callee = zsys_raw_sched_getaffinity;
```

Comment:
what's the difference between raw_sched_getaffinity (i.e. doing the syscall using syscall(2)) and sched_getaffinity (i.e. calling the libc function)?

Is it just the memseting of the difference between passed size and the result returned?